CSharpBashInterpreter/Commands/Basic/CatCommandRepresentation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CSharpBashInterpreter.Commands.Abstractions;
+
+namespace CSharpBashInterpreter.Commands.Basic;
+
+/// <summary>
+/// Representation for bash cat command
+/// </summary>
+public class CatCommandRepresentation : BaseCommandRepresentation
+{
+    public override string Name => "cat";
+
+    public override ICommandExecutable Build(IEnumerable<string> tokens)
+    {
+        return new CatCommandExecutable(tokens);
+    }
```

Comment:
```suggestion
    public override ICommandExecutable Build(IEnumerable<string> tokens)
        => new CatCommandExecutable(tokens);
```
Так просто писать короче

Reply:
Поправили в  #14 в рамках Задания №3